integration_tests/test_gravity.py

```diff
@@ -441,51 +452,35 @@ def test_gravity_cancel_transfer(gravity):
         amount = 1000
 
         print("send to cronos crc21")
-        recipient = HexBytes(ADDRS["community"])
-        send_to_cosmos(gravity.contract, erc20, recipient, amount, KEYS["validator"])
+        community = HexBytes(ADDRS["community"])
+        send_to_cosmos(gravity.contract, erc20, community, amount, KEYS["validator"])
         assert erc20.caller.balanceOf(ADDRS["validator"]) == balance - amount
 
         denom = f"gravity{erc20.address}"
-
         crc21_contract = None
 
-        def check_auto_deployment():
-            "check crc21 contract auto deployed, and the crc21 balance"
+        def local_check_auto_deployment():
             nonlocal crc21_contract
-            try:
-                rsp = cli.query_contract_by_denom(denom)
-            except AssertionError:
-                # not deployed yet
-                return False
-            assert len(rsp["auto_contract"]) > 0
-            crc21_contract = cronos_w3.eth.contract(
-                address=rsp["auto_contract"], abi=cronos_crc21_abi()
+            crc21_contract = check_auto_deployment(
+                cli, denom, cronos_w3, community, amount
             )
-            return crc21_contract.caller.balanceOf(recipient) == amount
-
-        def get_id_from_receipt(receipt):
-            "check the id after sendToChain call"
-            for _, log in enumerate(receipt.logs):
-                if log.topics[0] == HexBytes(
-                    abi.event_signature_to_log_topic(
-                        "__CronosSendToChainResponse(uint256)"
-                    )
-                ):
-                    return log.data
-            return "0x0000000000000000000000000000000000000000000000000000000000000000"
-
-        wait_for_fn("send-to-crc20", check_auto_deployment)
-
-        def check_fund():
-            v = crc21_contract.caller.balanceOf(ADDRS["community"])
-            return v == amount
+            return crc21_contract
```

Comment:
```suggestion
            return crc21_contract.caller.balanceOf(recipient) == amount
```

It should return a boolean here?

Reply:
need return contract back for [test](https://github.com/mmsqe/cronos/blob/bb9ffe112678f9e0b6c33ebdf289108b78501ba2/integration_tests/test_gravity.py#L273) to use